packages/opencode/precache-aft.py

```
@@ -0,0 +1,120 @@
+from datetime import datetime, timezone
+
+ORT_VERSION = sys.argv[1] if len(sys.argv) > 1 else None
+if not ORT_VERSION:
+    raise SystemExit("Usage: precache-aft.py <ort-version>")
+REPO = "Qdrant/all-MiniLM-L6-v2-onnx"
+FILES = ["model.onnx", "tokenizer.json"]
+HF = "https://huggingface.co"
+
+
+def fetch(url):
+    req = urllib.request.Request(url, headers={"User-Agent": "personal-setup-build"})
```

Ruff (0.16.1)

[error] Audit URL open for permitted schemes. Allowing use of `file:` or custom schemes is often unexpected.

(S310)